cloud/aws/templates/aws_oidc/bin/awsSetupTemplate.py

```diff
@@ -0,0 +1,46 @@
+import os
```

Comment:
camel case looks really out of place here, but I don't know if that's how it's supposed to be?

Reply:
okay yeah changed it